src/common/base/ConcurrentLRUCache.h

```diff
@@ -206,19 +206,21 @@ class LRU {
     }
 
     void insert(key_type&& key, value_type&& value) {
-        typename map_type::iterator i = map_.find(key);
-        if (i == map_.end()) {
-            // insert item into the cache, but first check if it is full
-            if (size() >= capacity_) {
-                VLOG(3) << "Size:" << size() << ", capacity " << capacity_;
-                // cache is full, evict the least recently used item
-                evict();
-            }
-            VLOG(3) << "Insert key " << key << ", val " << value;
-            // insert the new item
+        // insert item into the cache, but first check if it is full
+        if (size() >= capacity_) {
```

Comment:
what if I have a 10 capacity LRU, 
which I have key 1-10 in it. (list 1 to 10)
Then if I insert a key 5,
it will pop 10, which is not really necessary

Reply:
make sense